### PR TITLE
Fixed Slideshow

### PR DESCRIPTION
**Pull request by watakandai**
Hi John,

I fixed the slide show bug. 
I simply used the default method in jekyll which is to use `carousel.html` 

Could you check if it's working on your end (firefox)?

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 
+title:
 layout: default
 ---
 
-{% include slideshow.html images="/assets/images/main_carousel/0.jpg,/assets/images/main_carousel/1.png,/assets/images/main_carousel/2.png,/assets/images/main_carousel/3.jpg,/assets/images/main_carousel/4.png,/assets/images/main_carousel/5.jpg"%}
+{% include carousel.html height="50" unit="%" duration="7" %}
```
Comment (Jmcjack) [inline index.md:6]: Would like to pass in image references here so we can reuse carousel.html elsewhere

**Comment by Jmcjack** [comment]
👍 it works on FF, Safari, Brave and Chrome on my end. 

Only one suggestion - can you see if you can pass in the image references somehow to the carousel? I'd like to use this in multiple places.

**Comment by watakandai** [comment]
I can look into it, now.

**Comment by watakandai** [comment]
fixed!

**Comment by watakandai** [comment]
oh yeah ... I'll figure out why this happens.

**Comment by watakandai** [comment]
Done!

**Pull request merged**